src/components/FileBrowser.vue

```diff
@@ -0,0 +1,280 @@
+<script>
+import { sizeFormatter } from '../utils/mixins';
+import GirderBreadcrumb from './Breadcrumb.vue';
+
+const GIRDER_FOLDER_ENDPOINT = 'folder';
+const GIRDER_ITEM_ENDOINT = 'item';
+const ICON_MAP = {
+  folder: 'folder',
+  item: 'file',
+};
```

Comment:
This is meant to be captured by the icon mapping set up in [vuetifyConfig.js](https://github.com/girder/girder_web_components/blob/3f5ae62664e9f6fa02dbc451e6ccca3c19c1f81b/src/utils/vuetifyConfig.js#L8-L24), so this map feels redundant with that one.

Reply:
This currontly maps from girder `_modelType` to an icon described in vuetifyConfig.

My motivation here was to allow us to switch the icon mappings without having to refactor other parts of the code that may happen to use the same icons.

I believe it may be useful to separate our global icon map from the "filetype" map, which will probably get quite long as we add icons for images, xml, json, binary, and whatever else.  In the future, I think this map will also include mime types or extensions.

That being said, this may also be YAGNI.

Reply:
Is the intent that an *item* in this widget will be shown with different icons based on data type of the item? We don't formally encode the data type of an item like we do with files, so we'd need to come up with a system of rules to map an item to an icon based on some metadata in the item.

I think until we come up with that, it's probably premature to have a second level of indirection since we are only dealing with folder and item, which should be in the general icon map.